.circleci/config.yml

```diff
@@ -432,20 +409,30 @@ steps-electron-build-for-publish: &steps-electron-build-for-publish
     # ffmpeg
     - *step-ffmpeg-gn-gen
     - *step-ffmpeg-build
+    - *step-ffmpeg-store
 
     # Publish
-    - *step-electron-publish
-
+    - run:
+        name: Publish Electron Dist
```

Comment:
This step in not shared, so it can be inlined. And this way we can get fid of the `ELECTRON_RELEASE` check.